src/PicOfTheDay/PicOfTheDay.tsx

```diff
@@ -0,0 +1,45 @@
+import React, {useState, useEffect } from 'react';
+import ReactPlayer from "react-player"
+
+function PicOfTheDay(props) {
+    const[myData,setMyData] = useState(null);
+
+    useEffect(()=>{
+        fetch ("https://api.nasa.gov/planetary/apod?api_key=AdX53bpIZLYgz7TdNnGiGVcOGGGttMPnLhni9LxO ")
```

Comment:
As you are the first group to submit a PR with an API_KEY, we'll need to think about where we want to store an API key. We won't want to push our API keys to github and have them public (although these ones aren't very secret from NASA they are only used for rate limiting!) so we'll want a way to be able to use an API_KEY locally while not actually committing them to the project. I would suggest creating a .env file in the root folder, adding the API key there and using it as a variable in the code, and adding .env files to your .gitignore file so that it is not committed to the github repo. We should then update the ReadMe to explain that everyone will need to create their own .env file with their own API key

Reply:
By the root folder, do you mean the src folder containing all our component files, or the overarching repo folder?

Reply:
the latter so the repo folder :)

Reply:
https://create-react-app.dev/docs/adding-custom-environment-variables/ this should give some more information on how we are doing this